python/tvm/relay/frontend/pytorch.py

```diff
@@ -2485,6 +2509,12 @@ def interpolate(self, inputs, input_types):
         )
 
     def numel(self, inputs, input_types):
+        shape = self.infer_shape(inputs[0])
```

Comment:
We may want to check if we still need this change here. This is to fix a legacy bug in the old python impl.